docs/tutorials/Aragon_Core_User_Guide.md

```diff
@@ -0,0 +1,197 @@
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daoaddmember.png)
+
+**Step 2. Create vote to assign tokens**
+
+Even though you are the only tokenholder, the Democracy DAO still requires a tokenholder vote for every action. Click “Sign Transaction” to sign the transaction that creates a new vote to assign the new token you want to mint. 
+
+Note: Creating a vote will currently automatically make your account vote “Yes” on the proposal. You can change your vote by opening the vote in the Voting app, clicking the “No” button, and signing the transaction to change your vote to “No”.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daoaddmembervote.png)
+
+**Step 3. Sign and send transaction to create the “Assign Tokens” vote**
+
+After clicking “Sign Transaction”, a notification will pop-up in MetaMask. Open MetaMask, then sign and send the transaction to create the vote.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daoaddmembervotesign.png)
+
+Step 4. Check out the newest member of your DAO!
+
+Since you are the only tokenholder, the vote will automatically pass and the token automatically assigned to the account you added in Step 1. A good start to a new Democracy DAO!
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daoaddmemberapprove.png)
+
+#### Voting app
+
+The voting app makes it easy for you to poll your fellow DAO tokenholders about their position on an infinite range of topics. Votes initiated by other Aragon apps also show up in the Voting app. For example, assigning tokens using the Token Manager or moving funds from the Finance app are each actions that require a vote by tokenholders in a Democracy DAO.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daovoting.png)
+
+Follow these steps to create a new vote and poll your community about a topic you care about:
+
+**Step 1. Click “New Vote”**
+
+Fill out the question you want to poll your community about and click “Begin Vote”.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daonewvote.png)
+
+**Step 2. Sign transaction to create new vote**
+
+Click the “Sign Transaction” button, then open your MetaMask wallet to sign and send the transaction.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daonewvotesign.png)
+
+**Step 3. Ask the other DAO token-holders to review and vote on the proposal**
+
+Rally your fellow DAO tokenholders outside the app - for example, by email, forum, chat room, or good old fashioned face-to-face - and ask them to review and vote on the proposal. Tokenholders can visit the Vote app to see the list of open votes and cast their ballot accordingly.
```

Comment:
For example, link to https://aragon.chat on `chat room` here would be a good addition

Reply:
Not really, since DAOs will have their own chat rooms presumably.